specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -92,38 +91,40 @@ objects.
 
 === Transaction Requirements
 
-The Java EE platform must meet the following
+The Jakarta EE platform must meet the following
 requirements:
 
-* The Java EE platform must provide an object
+* The Jakarta EE platform must provide an object
 implementing the _javax.transaction.UserTransaction_ interface to all
 web components. The platform must publish the _UserTransaction_ object
 in the Java™ Naming and Directory Interface (JNDI) name space available
 to web components under the name _java:comp/UserTransaction_ .
-* The Java EE platform must provide classes
-that implement the transactional interceptors, as defined by the JTA
-specification.
+* The Jakarta EE platform must provide classes
+that implement the transactional interceptors, as defined by the Jakarta
+Transactions specification.
 * If a web component invokes an enterprise bean
-from a thread associated with a JTA transaction, the Java EE platform
-must propagate the transaction context with the enterprise bean
-invocation. Whether the target enterprise bean will be invoked in this
-transaction context or not is determined by the rules defined in the EJB
-specification.
+from a thread associated with a transaction defined by the Jakarta Transactions
+specification, the Jakarta EE platform must propagate the transaction context
+with the enterprise bean invocation. Whether the target enterprise bean will be
+invoked in this transaction context or not is determined by the rules defined in
+the Jakarta Enterprise Beans specification.
 
 Note that this transaction propagation
 requirement applies only to invocations of enterprise beans in the same
-Java EE product instancelink:#a3649[2] as the invoking component.
-Invocations of enterprise beans in another Java EE product instance (for
-example, using the EJB interoperability protocol) need not propagate the
-transaction context. See the EJB specification for details.
+Jakarta EE product instancelink:#a3649[2] as the invoking component.
```

Comment:
I don't think this anchor exists?

Reply:
Yes, this anchor does not exist. I removed it now.